### DISCUSSION TITLE
Discuss Project LUDO

### DISCUSSION THREAD

**Discussion by PAWEENNUTpuyfai** [MindIssawara/Cpp-Project#20] [General]
มาคุยเรื่องโปรเจ็คกันในนี้้นะะ

**Comment by PAWEENNUTpuyfai** [comment]
@l3eluGA @nnrwp เราสร้างไฟล์สำหรับลงfunction find_enemy ให้แล้วนะเพราะงั้นอัพตัวของFunction ที่พวกแกทำลงในไฟล์ find_enemy.hนะ

**Comment by MindIssawara** [comment]
ทุกคนว่าทำเกมเล่นได้ละหาเพลงมาใส่ดีมั้ย

**Comment by PAWEENNUTpuyfai** [comment]
@nnrwp @l3eluGA เราของfunction search_enemyภายในวันที่4นะ

**Comment by PAWEENNUTpuyfai** [comment]
ถ้าสีเขียวกับสีเขียวซ้อนกันอยู่แล้วสีแดงเข้ามาทับต้องไล่กลับท้งคู่ไหม

**Reply by MindIssawara** [reply]
เออวะ ไม่เคยเจอเลย....

**Reply by MindIssawara** [reply]
ไล่มันไปทั้งคู่กลับบ้านเลยละกัน

**Reply by PAWEENNUTpuyfai** [reply]
จัดไป!!!
